### ISSUE TITLE
CLI: CLI Event Streaming Tool for remote core (futurenet)

### ISSUE THREAD

**Issue by paulbellamy** [stellar/soroban-tools#61]
Extend the cli event streaming tool from #39 to work with a remote core (e.g. futurenet).

This depends on https://github.com/stellar/go/issues/4554 from soroban rpc first to provide the service endpoint.

- [x] Allow filtering by contract id

### Nice-to-haves

- [x] Allow filtering by other parts of the topic
- [ ] Allow config to push events to a message broker (e.g. sqs?)
  - How would devs like to use this? Maybe package it as a docker image with ENV-var flags?

**Comment by paulbellamy** [comment]
Blocked on: https://github.com/stellar/soroban-cli/issues/39

**Comment by jcx120** [comment]
For now, output will be in console.

**Comment by Shaptic** [comment]
Closed by #298.
